### PR TITLE
Usage of NameOf and String Interpolation.

### PR DESCRIPTION
**Pull request by AdamSpeight2008**

### CODE REVIEW

**Comment by AdamSpeight2008** [comment]
@dotnet-bot retest this please

File: src/Microsoft.VisualStudio.Editors/Common/ShellUtil.vb [hunk 1]
```diff
@@ -187,7 +189,7 @@ Namespace Microsoft.VisualStudio.Editors.Common
             Try
                 ProjAndSolutionProperties = DTE.Properties(EnvironmentCategory, ProjectsAndSolution)
                 If ProjAndSolutionProperties IsNot Nothing Then
-                    ShowValue = CBool(ProjAndSolutionProperties.Item("ShowAdvancedBuildConfigurations").Value)
```
Comment (davkean) [inline src/Microsoft.VisualStudio.Editors/Common/ShellUtil.vb]: Please revert this.

**Comment by davkean** [comment]
Please undo the lines where you have marked with "TODO". Please undo the Item lookup where you've changed behavior and broken simple configuration mode.

**Comment by davkean** [comment]
@AdamSpeight2008 Sorry for the late reply. This has merge conflicts, please resolve them - rebase and we'll merge this in.

**Pull request closed without merging**